x/pocketcore/keeper/claim.go

```diff
@@ -45,6 +49,39 @@ func (k Keeper) SendClaimTx(ctx sdk.Ctx, keeper Keeper, n client.Client, node *p
 			}
 			continue
 		}
+		if validator != nil && pc.GlobalPocketConfig.NotClaimPossiblyNegativeRewards {
+			rewardExpected, _ := k.posKeeper.CalculateRelayReward(
```

Comment:
The godoc for this function is:
```
// CalculateRelayReward - Calculate the amount of rewards based on the given
// number of relays and the staked tokens.  The returned reward amount includes
// DAO & Proposer portions.
```

Thoughts on renaming `rewardExpected` to a more semantic representation?

Reply:
The first variable returned here is the expected amount of the servicer's reward, so I think this name is correct.  Updated the leading comment of `CalculateRelayReward`.